.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -32,11 +34,19 @@ COPY library_generation /src
 # install protoc
 WORKDIR /protoc
 RUN source /src/utils/utilities.sh \
-	&& download_protoc "${PROTOC_VERSION}" "linux-x86_64"
+	&& download_protoc "${PROTOC_VERSION}" "${OS_ARCHITECTURE}"
 # we indicate protoc is available in the container via env vars
 ENV DOCKER_PROTOC_LOCATION=/protoc
 ENV DOCKER_PROTOC_VERSION="${PROTOC_VERSION}"
 
+# install grpc
+WORKDIR /grpc
+RUN source /src/utils/utilities.sh \
```

Comment:
`source /src/utils/utilities.sh` is not needed anymore. We should probably split
```
RUN source /src/utils/utilities.sh \
	&& download_protoc "${PROTOC_VERSION}" "${OS_ARCHITECTURE}"
```
to 
```
RUN source /src/utils/utilities.sh
RUN download_protoc "${PROTOC_VERSION}" "${OS_ARCHITECTURE}"
```
Then here we can just do 
```
RUN download_grpc_plugin "${GRPC_VERSION}" "${OS_ARCHITECTURE}"
```

Reply:
This is something I considered initially when working on the Dockerfile. The effects of `source` are contained in the `RUN` statement only, because it is translated into `/bin/sh -c "source script.sh"` (see [SO](https://stackoverflow.com/questions/20635472/using-the-run-instruction-in-a-dockerfile-with-source-does-not-work))

Reply:
Cool, that's good to know!